pkgs/games/koules/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, lib, fetchurl, fetchzip, makeDesktopItem, imake, gccmakedep, libX11, libXext, installShellFiles }:
+
+let
+  desktopItem = makeDesktopItem {
+    desktopName = "Koules";
+    name = "koules";
+    exec = "xkoules";
+    icon = "koules";
+    comment = "Push your enemies away, but stay away from obstacles";
+    categories = [ "Game" "ArcadeGame" ];
+  };
+  debian = fetchzip {
+    url = "mirror://debian/pool/main/k/koules/koules_1.4-27.debian.tar.xz";
+    sha256 = "0bq1rr6vxqmx2k0dhyrqnwwfiw4h2ycbj576v66vwr0jaq5plil3";
+  };
+in
+
+stdenv.mkDerivation rec {
+  pname = "koules";
+  version = "1.4";
+
+  src = fetchurl {
+    url = "https://www.ucw.cz/~hubicka/koules/packages/${pname}${version}-src.tar.gz";
+    sha256 = "06x2wkpns14kii9fxmxbmj5lma371qj00hgl7fc5kggfmzz96vy3";
+  };
+
+  # Debian maintains lots of patches for koules. Let's include all of them.
```

Comment:
Just out of curiousity, this is the only way to obtain those patches?

Reply:
Yes, I believe so. Debian does a good job keeping otherwise-unmaintained, but still nice, packages alive!